src/services/getUser.js

```diff
@@ -0,0 +1,11 @@
+const request = path => {
+  return fetch(`https://cors-anywhere.herokuapp.com/https://api.github.com/users/${path}`)
+    .then(res => Promise.all([res.ok, res.json()]))
+    .then(([ok, json]) => {
+      if(!ok) throw json;
+
+      return json;
+    });
+};
+
+export const getUser = user => request(user);
```

Comment:
it looks like `request` takes a path, but it seems like you return `request` with a `user` instead, on line 11. That's kinda confusing